event.json

```diff
@@ -0,0 +1,14 @@
+{
+  "pull_request": {
+    "number": 1,
```

Comment:
Consider using a more descriptive pull request number if this is part of a larger project. This will help in tracking changes and understanding the context of this PR.